NuKeeper/Program.cs

```diff
@@ -27,7 +28,12 @@ public static int Main(string[] args)
             var repositoryDiscovery = new GithubRepositoryDiscovery(github, settings);
             var updateSelection = new PackageUpdateSelection(settings.MaxPullRequestsPerRepository);
 
-            RunAll(repositoryDiscovery, lookups, updateSelection, github)
+            // get some storage space
+            var tempDir = TempFiles.MakeUniqueTemporaryPath();
+            var githubUser = github.GetCurrentUser().GetAwaiter().GetResult();
```

Comment:
I would prefer to push these lines into a single async method so that there's only one point of resynchronisation with `.GetAwaiter().GetResult()`